### PR TITLE
Release IIB v7.0.0

### PR DESCRIPTION
**Pull request by yashvardhannanavati**
Refers to CLOUDDST-17259

### CODE REVIEW

**Review by chandwanitulsi — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## 7.0.0
+- Fix flake8 and mypy bugs
+- Add new fbc-operations API endpoint
```
Comment (lipoja) [inline CHANGELOG.md:11]: no API tweaks mentioned :disappointed:

Comment (chandwanitulsi) [reply CHANGELOG.md:11]: Oh.. Thanks Jan

Comment (yashvardhannanavati) [reply CHANGELOG.md:12]: nice catch! Updating it

Comment (yashvardhannanavati) [reply CHANGELOG.md:12]: Done

Comment (lipoja) [reply CHANGELOG.md:12]: Thank you \o/

**Review by lipoja — APPROVED** [review]

**Pull request merged**